game/player.py

```diff
@@ -5,10 +5,14 @@
 
 
 class Player:
+    """Holds current information about the player"""
+
     team: List[Pokemon] = []
 
     def __init__(self):
-        if (len(self.team) < 1):
+        """fills it with a random Pokemon from PokeApi"""
```

Comment:
What is `it`? Also minor: capitalization

Reply:
Changed to `Fills "Team" array with a random Pokemon from PokeApi`

Reply:
Minor: don't capitalize references to variables.
`Fills the player's team with a random Pokemon from PokeApi`
The Player class doesn't actually do any of the saving to the global PLAYER variable, so I wouldn't include that.

Reply:
Changed to `Fills the player's team with a random Pokemon from PokeApi`